frontend/src/test/scala/bloop/tasks/CompilationHelpers.scala

```diff
@@ -17,7 +17,7 @@ object CompilationHelpers {
 
   private val ScriptedResolverId = "zinc-scripted-local"
   private val ScriptedResolveCacheDir: File =
-    AbsolutePath(s".ivy2/$ScriptedResolverId").toFile
+    AbsolutePath(s"../.ivy2/$ScriptedResolverId").toFile
```

Comment:
Can you add a comment explaining that the reason we need this is because `fork in Test` changes the working directory?